src/dev/ci_setup/setup.sh

```diff
@@ -5,6 +5,14 @@ set -e
 dir="$(pwd)"
 cacheDir="${CACHE_DIR:-"$HOME/.kibana"}"
 
+###
+### Since the Jenkins logging output collector doesn't look like a TTY
+### Node/Chalk and other color libs disable their color output. But Jenkins
+### can handle color fine, so this forces https://github.com/chalk/supports-color
+### to enable color support in Chalk and other related modules.
+###
+export FORCE_COLOR=1
```

Comment:
__question:__ hmm, where `FORCE_COLOR` is actually used? I mean you set it here, but seems to unset everywhere else (`jenkins_docs.sh`, `jenkins_selenium.sh` and `jenkins_unit.sh`) except for `jenkins_xpack.sh` or just don't understand how it works? :)

Reply:
> It's disabled for the single yarn bin call as yarn is supposed to print out just the location of the bin directory but with FORCE_COLOR=1 it also prints out some ansi control sequences that get embedded in the path and break things.

Interesting, thanks for clarifications, maybe mention this somewhere?